pkg/db/db.go

```diff
@@ -90,7 +100,7 @@ func Init(dbDir string, opts ...Option) (err error) {
 		debug.SetPanicOnFault(false)
 	}()
 
-	db, err = bolt.Open(dbPath, 0600, dbOptions.boltOptions)
+	db, err = bolt.Open(dbPath, dbOptions.mode, dbOptions.boltOptions)
```

Comment:
We don't have a case where we want to configure the permission. What if we just pass 0644 here?

Reply:
This `Init` method [is invoked by trivy](https://github.com/aquasecurity/trivy/blob/516e7cb906acb44d0d85f4e7bb123b17aab352a8/pkg/db/db.go#L36) so I wanted to avoid modifying the default behavior unless I had to. However, looking into it this wouldn't even result in the creation of a new `trivy.db` file, it's only used to open an existing (downloaded) one so the mode is irrelevant. If you'd prefer the simpler 1-line change to `bolt.Open` I can update the PR

Reply:
Yes, as you pointed out, Trivy doesn't create any new files. It's all done in GitHub Actions of trivy-db.
@DmitriyLewen Any concerns about permission updates?

Reply:
hm... I can't think of any issues for these changes.

Reply:
I've updated the PR to just directly modify this line to use `0644` by default

Reply:
didn't have time to write (you already merged this PR):
IIUC we also need to change perms here:
https://github.com/aquasecurity/trivy-db/blob/a2f579001fbd352f5882990872e386fa39a2484d/pkg/db/db.go#L88

Reply:
https://github.com/aquasecurity/trivy-db/pull/472